frontend/app/mappers/benefit-application-service-mappers.server.ts

```diff
@@ -217,14 +310,67 @@ function toRelatedPerson({ dateOfBirth, firstName, lastName, personRelationshipC
       },
     ],
     PersonRelationshipCode: {
-      ReferenceDataName: personRelationshipCode,
+      ReferenceDataName: 'Spouse',
     },
     PersonSINIdentification: {
       IdentificationID: socialInsuranceNumber,
     },
+    ApplicantDetail: {
+      ConsentToSharePersonalInformationIndicator: confirm,
+      AttestParentOrGuardianIndicator: undefined,
+      PrivateDentalInsuranceIndicator: undefined,
```

Comment:
Remove those two props since we don't have any information for those with a `spouse`.

Reply:
Reworked RelatedPerson type/function so having undefined types is no longer nessasary (removed the fields)